phase1/openstack/README.md

```diff
@@ -0,0 +1,45 @@
+# Getting started on OpenStack
+
+Prerequisites:
+* docker-engine which can be downloaded [here](https://docs.docker.com/engine/installation/)
+* working OpenStack deployment with: Keystone v3, Nova and Neutron (with LBaaS v2)
+
+Clone the kubernetes-anywhere repository.
+
+```console
+$ git clone https://github.com/kubernetes/kubernetes-anywhere.git
+$ cd kubernetes-anywhere
+```
+
+Hop into deployment shell which includes all the tools you need to deploy a kubernetes-anywhere cluster. Run:
+
+```console
+$ make docker-dev
+```
+
+If you don't already have an SSH key pair, you can generate one like below. SSH key pair is needed as a config during subsequent steps:
```

Comment:
Is there a way to specify this before jumping into the docker container's shell? Otherwise you need to move an existing key from your machine to the proper ``/opt/kubernetes-anywhere`` path beforehand.

Reply:
There could be multiple ways do it. 
1. One of the way could be just store the keys in project root directory. and once you move to container environment, it would automatically available.
2. Use docker volumes to make the keys available in docker container.